BO4E-dotnet.Encryption/BO4E.Encryption.csproj

```diff
@@ -13,18 +12,19 @@
     <Company>Hochfrequenz Unternehmensberatung GmbH</Company>
     <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
     <LangVersion>9</LangVersion>
-    <PackageVersion>0.2.7</PackageVersion>
```

Comment:
woher wird die info denn jetzt gezogen?

Reply:
Das erklärt der Onkel in dem Link, den ich Dir geschickt habe zu dem Thema. Default ist: Die PackageVersion wird aus der Version und die Version wird aus dem VersionPrefix und dem VersionSuffix zusammengesetzt. Der Version Suffix wird beim `dotnet pack` gesetzt.